.gitignore

```diff
@@ -2,8 +2,7 @@
 # Created by https://www.gitignore.io/api/rust,jupyternotebooks
 # Edit at https://www.gitignore.io/?templates=rust,jupyternotebooks
 
-### VScode ###
-.vscode/
+/
```

Comment:
not sure it's a good idea / useful to add `/` to .gitignore

Reply:
Just copy the full output of the link I provided (including update of comment,...)

Reply:
copy/pasted the generated https://www.gitignore.io/api/rust,jupyternotebooks,visualstudiocode